components/match2/commons/match_group_input_util.lua

```diff
@@ -1432,7 +1436,9 @@ function MatchGroupInputUtil.standardProcessFfaMaps(match, opponents, scoreSetti
 			map.winner = MatchGroupInputUtil.getWinner(map.status, winnerInput, map.opponents)
 		end
 
-		map.extradata = Parser.getExtraData and Parser.getExtraData(match, map, opponents) or nil
+		map.extradata = Parser.getExtraData and Parser.getExtraData(match, map, opponents) or {}
+		dateProps.date = nil -- don't need this in extradata
+		Table.mergeInto(map.extradata, dateProps)
```

Comment:
I think most ffa maps already set `dateexact` in their extradata. Seems redundant with this